doc/developers/contributing.rst

```diff
@@ -1325,6 +1325,27 @@ interface might be that you want to use it together with model evaluation and
 selection tools such as :class:`model_selection.GridSearchCV` and
 :class:`pipeline.Pipeline`.
 
+Setting `generate_only=True` allows for checks to be disassembled when
+testing an estimator. This feature can be used to parameterize tests in pytest
+as follows::
+
+  from itertools import chain
+  import pytest
+  from sklearn.utils.estimator_checks import check_estimator
+  from sklearn.utils.estimator_checks import set_check_estimator_ids
+  from sklearn.svm import LinearSVC
+  from sklearn.linear_model import LogisticRegression
+
+  estimators = [LinearSVC, LogisticRegression]
+
+  @pytest.mark.parametrize(
+      'estimator, check',
+      chain.from_iterable(check_estimator(est, generate_only=True)
+                          for est in estimators),
+      ids=set_check_estimator_ids)
+  def test_sklearn_compatible_estimator(estimator, check):
+      check(estimator)
```

Comment:
Currently this example is strictly equivalent to just calling `check_estimator` right? It would be useful to illustrate how to filter-out some tests (if that's the ultimate goal?)

Reply:
No, it's radically different and that should clearly be better documented.
if you call ``check_estimator``, it will fail on the first failing test and not run any other tests. This here will run all tests and report which ones are failing.

Reply:
Looks good to me apart from this. This difference should be explained and highlighted more!